src/sections/PartnersAndSponsorsSection/PartnersAndSponsorsSection.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import sponsers from "../../assets/data/sponsersData";
+import {gov,sponsors} from "../../assets/data/PastSponsors";
```

Comment:
And also instead of adding Two different data structures, why not just add the gov pictures in the sponsor's data itself? It would avoid confusion for the time being.